spring-kafka/src/main/java/org/springframework/kafka/support/micrometer/KafkaRecordSenderContext.java

```diff
@@ -56,5 +57,4 @@ public String getBeanName() {
 	public String getDestination() {
 		return this.destination;
 	}
-
```

Comment:
Blank lines between methods.
And therefore no changes in this class at all.
So, please, revert it to the `main` version.